manifests/c/CrystalDewWorld/CrystalDiskInfo/KureiKeiEdition/8.15.1/CrystalDewWorld.CrystalDiskInfo.KureiKeiEdition.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.0.6 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: CrystalDewWorld.CrystalDiskInfo.KureiKeiEdition
+PackageVersion: 8.15.1
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2022-02-11
+Installers:
+- Architecture: neutral
+  InstallerUrl: https://osdn.net/frs/redir.php?m=rwthaachen&f=crystaldiskinfo%2F76710%2FCrystalDiskInfo8_15_1KureiKei.exe
```

Comment:
```suggestion
  InstallerUrl: https://ftp.halifax.rwth-aachen.de/osdn/crystaldiskinfo/76710/CrystalDiskInfo8_15_1KureiKei.exe
```

Reply:
It does, it changed for each one of them, Levvie's request makes it auto like we want, just forgot to edit the url on these PRs